enhancements/sig-architecture/64-placementStrategy/README.md

```diff
@@ -0,0 +1,220 @@
+
+### Goals
+
+1. Add a new field placementStrategy to the placement APIs to let end uer define the roll out strategy for the selected clusters.
+
+### Non-Goals
+
+1. The new Placement APIs field placementStrategy has no control over applying the workloads.
+1. The new Placement APIs field placementStrategy has no control over scheduling the workloads.
+
+## Proposal
+
+### User Stories
+
+#### Story 1: As end user I want to define a subset of the selected clusters as a canary clusters.
+#### Story 2: As end user I want to divide the selected clusters to groups then a workloads applier APIs can use those cluster groups to roll out the workloads. 
+
+### Placement placementStrategy API
+
+Adding PlacementStrategy to PlacementSpec. PlacementStrategy has a labelSelector to identify canary clusters. CanaryClusters must be a subset of the selected clusters based on predicates and ClusterSets. ClustersGroupPercentage is a percentage of the total selected clusters that will divide the clusters to groups(batches). Example; for a total 100 clusters selected, clustersGroupPercentage equal to 20% will divide the selected clusters into 5 groups each group consist of 20 clusters.
+```go
+type PlacementSpec struct {
+	// ClusterSets represent the ManagedClusterSets from which the ManagedClusters are selected.
+	// +optional
+	ClusterSets []string `json:"clusterSets,omitempty"`
+
+	// Predicates represent a slice of predicates to select ManagedClusters. The predicates are ORed.
+	// +optional
+	Predicates []ClusterPredicate `json:"predicates,omitempty"`
+            ...
+ 
+	// PlacementStrategy define canary clusters as subset of the selected clusters and divide the selected clusters to groups.
+	// +optional
+	PlacementStrategy PlacementStrategy `json:"placementStrategy,omitempty"`
+}
+
+type PlacementStrategy struct {
+	// Label selector for canary clusters as subset of the selected clusters from predicates and ClusterSets.
+	// +optional
+	CanaryClusters metav1.LabelSelector `json:"canaryClusters,omitempty"`
+
+	// ClustersGroupPercentage is a percentage of the total selected clusters that will divide the clusters to groups.
+	// ex; for a total 100 clusters selected, ClustersGroupPercentage equal to 20% will divide the selected clusters to
+	//     5 groups each group should have 20 clusters.
+	// +kubebuilder:clustersGroupPercentage:Required
+	// +kubebuilder:clustersGroupPercentage:Enum=5%;10%;15%;20%;25%;30%;50%;100%
+	// +kubebuilder:default:=100%
+	// +optional
+	ClustersGroupPercentage string `json:"clustersGroupPercentage,omitempty"`
```

Comment:
How about MaxPercentage? I think `Max` describes the use of this field more accurately. The canary group and the last group will likely have less clusters. Same applies to the number of clusters per group if we plan to support that.